arangod/RestServer/DatabaseFeature.cpp

```diff
@@ -771,19 +766,18 @@ void DatabaseFeature::recoveryDone() {
 
   _pendingRecoveryCallbacks.clear();
 
-  auto databases = _databases.load();
+  if (!ServerState::instance()->isCoordinator()) {
```

Comment:
early return?

Reply:
Done

Reply:
Not here?) But it's NIT, so it's ok for me